### PR TITLE
MDBX geometry: match Erigon better

### PR DESCRIPTION
**Pull request by yperbasis**

### CODE REVIEW

File: db/silkworm/db/mdbx.cpp [hunk 1]
```diff
@@ -90,7 +92,10 @@ ::mdbx::env_managed open_env(const EnvConfig& config) {
     op.max_readers = config.max_readers;
 
     ::mdbx::env_managed ret{db_path.native(), cp, op, config.shared};
-    // TODO (Andrea) C++ bindings don't have setoptions
+
+    // C++ bindings don't have setoptions
+    ::mdbx::error::success_or_throw(::mdbx_env_set_option(ret, MDBX_opt_rp_augment_limit, 32 * 1024 * 1024));
+
```
Comment (AndreaLanfranchi) [inline db/silkworm/db/mdbx.cpp:98]: I'm not sure this is the correct way (i'm stil testing).
If you notice erigon's code you'll notice options are set **before** env is opened whilest the creation of instance of env_managed immediately opens the underlying data file

Comment (yperbasis) [reply db/silkworm/db/mdbx.cpp:98]: I see that `rp_augment_limit` is used in `mdbx_page_alloc()`, so it't probably OK to set it after the environment is created.

Comment (AndreaLanfranchi) [reply db/silkworm/db/mdbx.cpp]: Sorry probably we're not on the same page. Env created != Env opened.

Comment (yperbasis) [reply db/silkworm/db/mdbx.cpp]: What I mean is that mdbx_page_alloc() is called most of the time during normal operations, after the environment is created and opened, so it should be OK to set MDBX_opt_rp_augment_limit later.

**Review by AndreaLanfranchi — APPROVED** [review]

**Pull request merged**